packages/atomic/create-atomic/src/plopfile.ts

```diff
@@ -184,15 +184,20 @@ export default async function (plop: NodePlopAPI) {
           return new Promise((resolve, reject) => {
             const {project} = data as PlopData;
             const process = spawn(getPackageManager(), ['install'], {
-              stdio: 'ignore',
+              stdio: ['ignore', 'ignore', 'pipe'],
               cwd: join(currentPath, project),
             });
 
+            let error = '';
+            process.stderr.on('data', (chunk) => {
+              error += chunk.toString();
+            });
+
             process.on('close', (code) => {
```

Comment:
https://nodejs.org/api/child_process.html#event-close gimme the signal too. When reported, it's often a bliss in term of information (i.e. the signal is often very specific and so with it fixing a problem is often a breeze [^1])

[^1]: famouslastwords.